packages/jest-emotion/src/replace-class-names.js

```diff
@@ -5,7 +5,7 @@ function defaultClassNameReplacer(className, index) {
 
 export type ClassNameReplacer = (className: string, index: number) => string
 
-const componentSelectorClassNamePattern = /e[a-zA-Z0-9-]+[0-9]+/
+const componentSelectorClassNamePattern = /^e[a-zA-Z0-9]+[0-9]+$/
```

Comment:
Where is the `'-'`? I mean, is it OK to remove `'-'`?
This pattern does not match with `emotion-0`....

Reply:
It's not meant to match `emotion-0`.

It's meant to match class names like this `e17ivhiq0`.

Component selector class names used to be like `css-17ivhiq0` so this included `-` and I just forgot to remove it when I changed the shape of the component selector class names

Reply:
Thank you for your detailed answer! :)

Reply:
@mitchellhamilton Wa...it, this also does not match with `e17ivhiq0`, does it?
It matches with `e17` or `eifjwe5239`, but does not with `e17ivhiq0`.

Reply:
Never mind. I'm the fool.